Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBCollection-find.txt

MongoDB\\Collection::find()
===========================

.. meta::

>    :description: Discover how to use `MongoDB\Collection::find()` to query documents with various options like filters, projections, and sorting in PHP.

Reviewer: ```suggestion
   :description: Query documents with the MongoDB PHP Library, with various options like filters, projections, and sorting in PHP.
```
